root_VS2010/programs/C#/Frameworks/Infrastructure/Framework/Util/FxHttpSessionIndex.cs

```diff
@@ -59,6 +60,9 @@ public static class FxHttpSessionIndex
         /// <summary>ボタン履歴を保持する「キュー」を格納するSessionキー</summary>
         public const string BUTTON_HISTORY = "ButtonHistory";
 
+        /// <summary> Session key that contains the dummy value. </summary>
+        public const string DUMMY = "dummy";
```

Comment:
@yusukemaegawa 

The root of the merge is as follows.
- root -> root_easysetup -> root_VS2010 -> root_VS2012 -> root_VS2013 -> root_VS2015

Changes of new version is required to merge to the old version because of following reasons.

If did not merge the changes to the old version,
- We can not verify conflict between the implementation of the functions of each other at the root folder.
- There is the possibility that will be misjudged to unmerged at the confirmation of the merge working with diff tool.